### PR TITLE
Update towards-reproducibility-pinning-nixpkgs.rst

### PR DESCRIPTION
**Pull request by arianvp**
Fix a typo and explain how you can change branch in niv

### CODE REVIEW

File: source/tutorials/towards-reproducibility-pinning-nixpkgs.rst [hunk 1]
```diff
@@ -59,11 +59,21 @@ dependencies are fetched. It will also create ``nix/sources.nix`` which glues th
 
 By default ``niv`` will use the **latest stable** NixOS release. However, you should check to see which version is currently specified in `the niv repository <https://github.com/nmattia/niv>`_ if you require a specific release, as it might lag behind.
 
+You can see which version ``niv`` is tracking as follow:
+
+    $ niv show
+
+And you can change the tracking branch to the one you want like this:
+
+    $ niv modify nixpkgs --branch < nixpkgs-unstable | nixos-20.03 >
```
Comment (domenkozar) [inline source/tutorials/towards-reproducibility-pinning-nixpkgs.rst:68]:
```suggestion
    $ niv modify nixpkgs --branch nixos-20.03
```

**Comment by domenkozar** [comment]
recheckcla

**Comment by domenkozar** [comment]
@arianvp you'll need to rebase and accept CLA :)

**Comment by arianvp** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by domenkozar** [comment]
recheckcla

**Pull request merged**